docs/example-api-calls/zoom.md

```diff
@@ -1,63 +1,52 @@
 # API Call Examples for Zoom
 
-Example commands that you can use to validate proxy behavior against the Zoom APIs.
+Example commands (*) that you can use to validate proxy behavior against the Zoom APIs.
 Follow the steps and change the values to match your configuration when needed.
 
-```shell
-export PROXY_ZOOM_URL=https://YOUR_PSOXY_HOST/psoxy-zoom
-```
+For GCP, you can use the `-i` flag to impersonate the desired user identity option when running the testing tool. Example:
 
-For GCP, use
 ```shell
-export OPTIONS="-g"
+node tools/test-psoxy.js -u [your_psoxy_url]/psoxy-gcal/calendar/v3/calendars/primary -i you@acme.com
```

Comment:
update to a zoom example (or drop these docs, rely on test suites / calls generated from Terraform)